router/router.go

```diff
@@ -109,6 +109,16 @@ func Load(options ...gin.HandlerFunc) *gin.Engine {
 
 		// Worker endpoints
 		WorkerHandlers(baseAPI)
+
+		// Pipeline endpoints
+		pipelines := baseAPI.Group("pipelines/:org/:repo", repo.Establish())
```

Comment:
should we wrap this in a function as other handlers are here? i think we have some other pretty minimal ones wrapped.

Reply:
yes absolutely! 👍  done!